cmd/operator-sdk/alpha/scorecard/cmd.go

```diff
@@ -15,33 +15,54 @@
 package scorecard
 
 import (
-	log "github.com/sirupsen/logrus"
-	"github.com/spf13/cobra"
-)
+	"fmt"
+	"log"
 
-var (
-	config   string
-	bundle   string
-	selector string
-	listAll  bool
+	scorecard "github.com/operator-framework/operator-sdk/internal/scorecard/alpha"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/labels"
 )
 
 func NewCmd() *cobra.Command {
+	var (
+		config   string
+		bundle   string // TODO - to be implemented
+		selector string
+		list     bool // TODO - to be implemented
+	)
 	scorecardCmd := &cobra.Command{
 		Use:    "scorecard",
 		Short:  "Runs scorecard",
 		Long:   `Has flags to configure dsl, bundle, and selector.`,
 		Hidden: true,
-		Run: func(cmd *cobra.Command, args []string) {
-			log.Info("TODO")
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			var err error
+			o := scorecard.Options{}
```

Comment:
Does this make sense:
```suggestion
			o := scorecard.Options{List: list}
```

Reply:
since list is not implemented and I've not made a decision on where the list code should go, I'd like to wait on this one for now.